mapic/stats_collector.go

```diff
@@ -87,11 +70,11 @@ func (c *metricsCollector) collectMetrics(ctx context.Context) error {
 			glog.Errorf("Error getting stream info for streamId=%s err=%q", streamID, err)
 			continue
 		}
-		if info.isLazy {
-			// avoid spamming metrics for playback-only catalyst instances. This means
-			// that if mapic restarts we will stop sending metrics from previous
-			// streams as well, but that's a minor issue (curr stream health is dying).
-			glog.Infof("Skipping metrics for lazily created stream info. streamId=%q metrics=%+v", streamID, metrics)
+
+		stream := c.infoProvider.wildcardPlaybackID(info.stream)
+		isIngest := isIngestStream(stream, info, mistState)
+
+		if !isIngest {
 			continue
```

Comment:
WDYT of adding a verbose log here ([`glog.V(8)...`](https://github.com/livepeer/livepeer-infra/blob/ae887391834954e4809a5086f816e69a6ccd0d87/values/staging/catalyst.values.yaml#L21)) when a stream is skipped, so you can explicitly test that on staging? i.e.: playback a stream from a different node and make sure that node doesn't send metrics, only the ingest node does.